docker/blocks/jaeger/fig

```diff
@@ -0,0 +1,6 @@
+jaeger:
+  image: jaegertracing/all-in-one:latest
+  ports:
+    - "localhost:5775:5775/udp"
```

Comment:
is this the right port?

Reply:
I used the same port as the quick guide http://opentracing.io/documentation/pages/quick-start

Looking at http://jaeger.readthedocs.io/en/latest/getting_started/ I guess not. I'll update to 6831 (accept jaeger.thrift over compact thrift protocol)